.travis/setup-kubernetes.sh

```diff
@@ -5,6 +5,18 @@ function install_kubectl {
     sudo cp kubectl /usr/bin
 }
 
+function install_registry {
+    sudo -E $TEST_CLUSTER addons enable registry
+    # Wait for the registry pod to be ready
+
+    until $(kubectl --namespace=kube-system get po $(kubectl get po -n kube-system | grep 'registry-[a-z0-9]' | awk '{print $1;}') --no-headers | grep -Eq '([1-9]+[0-9]*)/\1'); do
+        printf '.';
+        sleep 5;
+    done
+
+    kubectl port-forward --namespace kube-system $POD 5000:5000 &
```

Comment:
where the `$POD` var is set ?

Reply:
I was originally setting it like `POD=$(kubectl get po -n kube-system | grep 'registry-[a-z0-9]' | awk '{print $1;})`, which I was executing *before* the loop. The problem is that, initially when minikube starts it doesn't even know what pod resources are, so it went wrong before the loop was even entered. 

See the interesting discussion about why kubectl doesn't handle readiness already: https://github.com/kubernetes/kubernetes/issues/1899

Reply:
I'm really dumb but I still don't understand from where the `$POD` var comes from or where it's set :-)

Reply:
@ppatierno I updated the PR, so there is no $POD anymore